docs/docs/icechunk-python/dask.md

```diff
@@ -5,23 +5,16 @@ However, because of how Icechunk works, it's not possible to use the existing [`
 
 Instead, Icechunk provides its own specialized functions to make distributed writes with Dask and Xarray.
 This page explains how to use these specialized functions.
-!!! note
 
-    Using Xarray, Dask, and Icechunk requires `icechunk>=0.1.0a5`, `dask>=2024.11.0`, and `xarray>=2024.11.0`.
 
+Start with an icechunk store and dask arrays.
 
-First let's start a distributed Client and create an IcechunkStore.
-
-```python
-# initialize a distributed Client
-from distributed import Client
-
-client = Client()
-
```

Comment:
noteable change here is that the first example doesn't use a client, because it was not working without `allow_pickling` so i only used the client after that was described